Islandora/islandora-community
# Ignore .DS_Store files

# What does this Pull Request do?

Provide a brief description of what the intended result of the PR will be and/or what
 problem it solves.

This pull request simply ignores `.DS_Store` files introduced by some specific operating systems.

# What's new?

Introduced a `.gitignore` file to ignore future additions of the files in question.

# How should this be tested?

Check out this branch / pull request.
Open the working directory in an operating system such as MacOS
Inspect the directory structure for the addition of `.DS_Store` files
Confirm the files are ignored by Git, so they are no longer tracked by source control.

# Documentation Status

No documentation needed, `.gitignore` is self-documenting for the most part.

# Additional Notes:

Simply testing this Islandora engagement vector with a tiny yet useful change.

# Interested parties

Tag (@ mention) interested parties or, if unsure, @Islandora/committers -- Hello! 👋
